### ISSUE TITLE
Add notice of CC0 and imported source restrictions to Data Accessibility page

### ISSUE THREAD

**Issue by mcannon068nw** [dgidb/dgidb-v5#620]
The data accessibility page doesn't explicitly state that the data is available under CC0 license terms (except where imported sources have restrictions). This is an oversight and should be written out explicitly on the Data Accessibility page. 

(We could use the same important box styling from the citations to highlight this point towards the bottom)
